java/ql/test/library-tests/frameworks/spring/ValidationErrorsTest.java

```diff
@@ -0,0 +1,72 @@
+import org.springframework.validation.Errors;
```

Comment:
Looks like you forgot to commit a few files: the `.ql` and `.expected` files are missing here.